Iris/FrontEndTest/TestCompilerContext.cs

```diff
@@ -50,6 +50,11 @@ internal sealed class TestCompilerContext : CompilerContext
 
         public static TestCompilerContext Create(string compiland, GlobalSymbolList globals, CompilationFlags flags)
         {
+#if NETCOREAPP
+            flags |= CompilationFlags.NetCore;
+            flags |= CompilationFlags.WriteDll;
```

Comment:
>flags |= CompilationFlags.WriteDll; [](start = 12, length = 35)

Why is this needed for TestCompilerContext?  We should be writing text instead of a PE.

Reply:
Yup. Doesn't seem to be needed. I will remove it.